### DISCUSSION TITLE
Vertical Zoom

### DISCUSSION THREAD

**Discussion by lal12** [OpenHantek/OpenHantek6022#274] [Q&A] [answered]
I have a 24V signal with a small ripple (~0.5V). I would like to look at the ripple more closely, but it's pretty small since with 5V/div it is quite small on the screen. 

So is there either an offset feature to place the zero point out of visible range or can the zoom window also scale vertically?

**Comment by Vascom** [comment]
You can use capacitor. For example 10-100nF film will be good to remove constant component.

**Reply by lal12** [reply]
Ok, sure. But a Software feature like zooming in vertically as better DSOs can would still be preferable IMO.

**Comment by Ho-Ro** [comment, marked as answer]
You can also use the the MATH channel and select the function "CH1 AC" to remove the DC offset.

**Comment by Vascom** [comment]
If you take 1:10 probe with 8-bit openhanteks ADC it will be 50V/128 = 0.39V per one bit. It is very coarse to measure 0.5V ripple.

**Reply by Ho-Ro** [reply]
Ok, but with slow sample rates OH oversamples up to 100x, that can give 2 or 3 more bits when your signal is a little bit noisy - but it is always better to have AC coupling, either with your capacitor hack (or use a DC-blocker interface plug), or best with my [AC modification](https://github.com/OpenHantek/OpenHantek6022/blob/main/docs/HANTEK6022_AC_Modification.pdf).
